README.md

```diff
@@ -33,8 +33,8 @@ philosophy](https://abseil.io/about/philosophy).
 `prototype-kicad-library-ci` uses the [GitHub
 flow](https://guides.github.com/introduction/flow/) workflow.
 
-## Git Config Install Instructions
-To install this repository's Git configuration, execute the `install` script located in the
+## Git Config
+To install this repository's Git config entries, execute the `install` script located in the
 `git/config` directory.
```

Comment:
Please use a column limit of 90 (see https://github.com/apcountryman/prototype-kicad-library-ci/blob/main/CONTRIBUTING.md#style-guide).
```suggestion
To install this repository's Git config entries, execute the `install` script located in
the `git/config` directory.
```